src/commands/publish.rs

```diff
@@ -61,8 +61,19 @@ pub fn publish(
         // Next, upload and deploy the worker with the updated asset_manifest
         upload::script(&upload_client, &target, Some(asset_manifest))?;
 
-        deploy::worker(&user, &deploy_config)?;
-
+        if message::is_json_output() {
+            let result = build(&target);
+            let mut jsonoutput = message::PublishOutput {
+                name: Some(target.name.clone()),
+                success: match result {
+                    Ok(_) => Some(true),
+                    Err(_) => Some(true),
```

Comment:
```suggestion
                    Err(_) => Some(false),
```

Reply:
that's a big oof right there- thanks for catching

Reply:
haha thats what code review friends are for!